src/enforcement/enforcer.ts

```diff
@@ -143,6 +148,18 @@ export class Enforcer implements IEnforcer {
         },
       });
     }
+    if (config.opaAxiosInstance) {
+      this.opaClient = config.opaAxiosInstance;
+      this.opaClient.defaults.baseURL = opaBaseUrl.toString();
+      this.opaClient.defaults.headers.common['X-Permit-SDK-Version'] = `node:${version}`;
+    } else {
+      this.opaClient = axios.create({
+        baseURL: opaBaseUrl.toString(),
```

Comment:
That still is a too specific path for OPA. On the `opaClient` have it the root path and specify `/v1/data/permit` on the specific check call.

If the user / future development would want to request OPA directly, they will be able to reuse this client

Reply:
isn't it the prefix for all of the user operations ? check, userpermissions ,atuhorized_user etc ?